examples/TLS_DenseSkOp.cpp

```diff
@@ -0,0 +1,164 @@
+    RandBLAS::sketch_general<double>(
+            blas::Layout::ColMajor,    // Matrix storage layout of AB and SAB
+            blas::Op::NoTrans,         // NoTrans => \op(S) = S, Trans => \op(S) = S^T
+            blas::Op::NoTrans,         // NoTrans => \op(AB) = AB, Trans => \op(AB) = AB^T
+            sk_dim,                    // Number of rows of S and SAB
+            n+d,                       // Number of columns of AB and SAB
+            m,                         // Number of rows of AB and columns of S
+            1,                         // Scalar alpha - if alpha is zero AB is not accessed
+            S,                         // A DenseSkOp or SparseSkOp sketching operator
+            0,                         // Row offset of S
+            0,                         // Column offset of S
+            AB,                        // Matrix to be sketched
+            m,                         // Leading dimension of AB
+            0,                         // Scalar beta - if beta is zero SAB is not accessed
+            SAB,                       // Sketched matrix SAB
+            sk_dim                     // Leading dimension of SAB
+    );
+    auto time_sketch2 = high_resolution_clock::now();
+
+    // Perform SVD operation on SAB
+    auto time_TLS1 = high_resolution_clock::now();
+    lapack::gesdd(lapack::Job::AllVec, sk_dim, (n+d), SAB, sk_dim, svals, U, sk_dim, VT, n+d);
+            
+    for (int i = 0; i < n; i++) {
+        X[i] = VT[(n+d-1) + i*(n+d)]; // Take the right n by 1 block of V
+    }
+
+    // Scale X by the inverse of the 1 by 1 bottom right block of V
+    blas::scal(n, -1/VT[(n+d)*(n+d)-1], X, 1); 
+    auto time_TLS2 = high_resolution_clock::now();
+
+    //Check TLS solution. Expected to be a vector of 1's
+    double res_infnorm = 0;
+    double res_twonorm = 0;
+
+    for (int i = 0; i < n; i++) {
+        res[i] = abs(X[i] - 1);
+        res_twonorm += res[i]*res[i];
+        if (res_infnorm < res[i]) {
+            res_infnorm = res[i];
+        }
+    }
+
+    std::cout << "Matrix dimensions:                              " << m << " by " << n+d << '\n'; 
+    std::cout << "Sketch dimension:                               " << sk_dim << '\n'; 
+    std::cout << "Time to create dense sketch:                    " << (double) duration_cast<milliseconds>(time_constructsketch2 - time_constructsketch1).count()/1000 << " seconds" << '\n';
+    std::cout << "Time to sketch AB:                              " << (double) duration_cast<milliseconds>(time_sketch2 - time_sketch1).count()/1000 << " seconds" <<'\n';
+    std::cout << "Time to perform TLS on sketched matrix:         " << (double) duration_cast<milliseconds>(time_TLS2 - time_TLS1).count()/1000 << " seconds" << '\n';
+    std::cout << "Inf-norm of TLS residal vector:                 " << res_infnorm << '\n';
+    std::cout << "Two-norm of TLS residual vector:                " << sqrt(res_twonorm) << '\n';
```

Comment:
The vector ``X - expected_X`` isn't what's usually meant by "residual vector" in the context of  least squares (total, or otherwise).

Suggestion: report "inf-norm distance from TLS solution to vector of all ones" and "two-norm distance from TLS solution to vector of all ones".